cmd/run.go

```diff
@@ -164,6 +164,7 @@ To skip bundle verification, use the --skip-verify flag.
 
 	addConfigFileFlag(runCommand.Flags(), &cmdParams.rt.ConfigFile)
 	runCommand.Flags().BoolVarP(&cmdParams.serverMode, "server", "s", false, "start the runtime in server mode")
+	runCommand.Flags().DurationVarP(&cmdParams.rt.ReadyTimeout, "ready-timeout", "", 0, "wait for configured bundles and plugins before starting server (value <= 0 disables ready check)")
```

Comment:
nit: For consistency with `shutdown-grace-period` lets have this be an `int` flag where the value is in seconds. In retrospect it would have been nice to use duration for both... we will have an opportunity in the future to change them, but for now lets stick with number of seconds.